tools/aie2xclbin/CMakeLists.txt

```diff
@@ -17,22 +17,30 @@ llvm_map_components_to_libnames(llvm_libs support)
 target_link_libraries(aie2xclbin ${llvm_libs})
 
 get_property(dialect_libs GLOBAL PROPERTY MLIR_DIALECT_LIBS)
+get_property(conversion_libs GLOBAL PROPERTY MLIR_CONVERSION_LIBS)
+get_property(extension_libs GLOBAL PROPERTY MLIR_EXTENSION_LIBS)
 
 configure_file(configure.h.in configure.h)
 target_include_directories(aie2xclbin PRIVATE "${CMAKE_CURRENT_BINARY_DIR}")
 
 target_link_libraries(aie2xclbin
   ${dialect_libs}
+  ${conversion_libs}
+  ${extension_libs}
   MLIRParser
   MLIRSCFToControlFlow
   MLIRAffineToStandard
+  MLIRAIEVecDialect
+  MLIRAIEVecToLLVM
+  MLIRAIEVecTransformOps
```

Comment:
You might not need this particular one (`MLIRAIEVecTransformOps`), that's the `transform.structured` extension I added to vectorize matmul-like contractions in `linalg.generic` ops.

Reply:
Good point. I sprayed everything I could at this cmake file to get everything compiling and linking. I need to remove the unnecessary parts (like MLIRAIEVecTransformOps)